pkg/v2/service.go

```diff
@@ -343,6 +350,19 @@ func (s *service) Create(ctx context.Context, r *taskAPI.CreateTaskRequest) (_ *
 	// save the main task id and bundle to the shim for additional requests
 	s.id = r.ID
 	s.bundle = r.Bundle
+
+	// Set up OOM notification on the sandbox's cgroup. This is done on sandbox
+	// create since the sandbox process will be created here.
+	pid := process.Pid()
+	if pid > 0 {
+		cg, err := cgroups.Load(cgroups.V1, cgroups.PidPath(pid))
+		if err != nil {
+			logrus.WithError(err).Errorf("loading cgroup for %d", pid)
```

Comment:
return error?

Reply:
I went with what runc's shim does here and I think the logic is that not being able to report oom killer events isn't fatal, but looking at this now runc's shim and the code here both use cg even in the event of an error being returned from cgroups.Load so it's just another example of me blindly copying containerd code and thinking it's ok :(

Done.